README.md

```diff
@@ -76,6 +76,50 @@ For more info about the API, see the [API reference](https://kontent.ai/learn/re
 
 You can find the Delivery and other SDKs at <https://github.com/Kentico>.
 
+## About
+This section describes the application.
+
+### Used toolchain
+This application is based on the [Create React App](https://reactjs.org/docs/create-a-new-react-app.html) using the following template `--template typescript`.
+
+### Model mapping
+There are two types of model mapping in this application:
+- content types -> component
+- component -> view model
+
+For generating component models from content types, we have used [Kontent.ai model generator](https://github.com/Kentico/kontent-model-generator-js) tool. All generated models can be found in in [src/Models](https://github.com/Kentico/kontent-sample-app-react/tree/master/src/Models) folder. The `_project.ts` exports `projectModel` which contains information about the project structure such as project languages as well as information such as codenames about content types. Those generated models are used to obtain correctly typed objects via client in [src/Pages](https://github.com/Kentico/kontent-sample-app-react/tree/master/src/Pages). To obtain your data provide a generated DTO type into generic brackets for `items` method and content types codename from `projectModel` as a parameter of `type` method. See following example:
+
+```{typescript}
+const query = Client.items<generatedDTO>()
+      .type(projectModel.contentTypes.generatedDTO.codename)...
+```
+

```

Comment:
I think there might be a little misunderstanding. I do not think that we use stateless components in pages. Pages fetch data by themselves. Only example which I could find to use components pattern are `BrewerStoreContainer` with `BrewerStoreListing` and `CoffeStoreContainer` with `CoffeStoreListing`

Are those what you have in mind? Because comment below you have code sample of page which fetch data by itself.